CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+Changelog
+=========
+
+XXXXXXXXXX version 1.0.1:
+-------------------------
+
+  **Bug fixes**:
+
+    - Fix missing role settings when several roles exists.
```

Comment:
I guess a slightly more explaining message could be helpful for users.  How about  "Fix issue leading to duplicated role settings changes" or similar?